src/main/resources/db/changelog/changes/v1.0.0/_cleanup-rmb.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet id="MODPWD-64@@drop-old-validation-rules-table" author="psmagin">
+        <sql dbms="postgresql">DROP TABLE IF EXISTS validation_rules</sql>
```

Comment:
the table doesn't exist in mod-tags. this statement is not necessary
please remove